CONTRIBUTING.md

```diff
@@ -49,6 +49,24 @@ on how to start working on Servo.
 
 For specific git instructions, see [GitHub workflow 101](https://github.com/servo/servo/wiki/Github-workflow).
 
+## Running tests in pull requests
+
+When you push to a pull request, GitHub automatically checks that your changes have no compile, lint, or tidy errors.
+
+To run unit tests or Web Platform Tests against a pull request, you can mention [@bors-servo](https://github.com/bors-servo) in a comment, or add one or more labels to your pull request:
+
+| comment | label |
+|---|---|
+| `@bors-servo try`<br>`@bors-servo try=full` | `T-full` |
+| `@bors-servo try=wpt-2013`<br>`@bors-servo try=wpt`<sup>1</sup> | `T-linux-wpt-2013` |
+| `@bors-servo try=wpt-2020` | `T-linux-wpt-2020` |
+| `@bors-servo try=linux` | `T-linux-wpt-2020`<sup>2</sup> |
+| `@bors-servo try=macos` | `T-macos` |
+| `@bors-servo try=windows` | `T-windows` |
+
+1. this will become equivalent to `try=wpt-2020` at some point
+2. unlike `try=linux`, this runs WPT tests too, not just unit tests
+
```

Comment:
Would it be okay to hold off on this a bit? The hope is that the labels (which we're still testing) are good enough to remove the comment style entirely, but only if the GitHub Actions works better. The motivation is that there are quite a few issues with the GitHub Action that is run for every PR comment (lots of spurious failures).

Reply:
sure, no worries! closing for now, we can revisit this later :)

Reply:
now that #30712 is merged, how does this look with 9b28c527e6534afdfb40691f7b72d7135d34bb17?